packages/components/pages/f-offers/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v1.12.0-beta.3
+-----------------------------
+*July 12, 2022*
+
+### Changed
+- **breaking changes** Update to `@use` and `@forward` SASS syntax
+
```

Comment:
When the latest changelog entry is a `Latest` and you add another one, they should be merged together.

In other words, delete the `Latest` entry and add
```
- Refactor WebDriverIO tests to use async in order to support Node 16 using `codemod` utility.
```
to your `Changed` section